Account/FxAClient10.swift

```diff
@@ -237,36 +237,36 @@ public class FxAClient10 {
         mutableURLRequest.HTTPMethod = Method.POST.rawValue
 
         mutableURLRequest.setValue("application/json", forHTTPHeaderField: "Content-Type")
-        mutableURLRequest.HTTPBody = JSON(parameters).toString(pretty: false).utf8EncodedData
+        mutableURLRequest.HTTPBody = JSON(parameters).toString(false).utf8EncodedData
 
-        let request = alamofire.request(mutableURLRequest)
+        _ = alamofire.request(mutableURLRequest)
             .validate(contentType: ["application/json"])
-            .responseJSON { (request, response, data, error) in
-                if let error = error {
-                    deferred.fill(Result(failure: FxAClientError.Local(error)))
+            .responseJSON { (request, response, result) in
+                if let error = result.error as? NSError {
```

Comment:
Actually this looks like it's okay because Alamofire does encode it's errora as an NSError :+1: